src/backlight/backlight.c

```diff
@@ -226,26 +226,20 @@ static int validate_device(sd_device *device) {
 }
 
 static int get_max_brightness(sd_device *device, unsigned *ret) {
```

Comment:
hmm? This one i don't grok? the old code returned SYNTHETIC_ERRNO(EINVAL), not 0. What am I missing? Can't map your commit msg to the actual code change.

Not sure it's a big benefit to leave the validation to the caller, instead of encapuslating this in the function

Reply:
Oh, duh. I managed to somehow confuse myself and changed the text from the original version to this when pushing the patches. Fix coming right up.